app/views/foreman_ansible/job_templates/ansible_windows_updates.erb

```diff
@@ -0,0 +1,187 @@
+  hidden_value: false
+- name: reboot
+  required: false
+  input_type: user
+  description: "Ansible will automatically reboot the remote host if it is required
+    and continue to install updates after the reboot.\r\nThis can be used instead
+    of using a win_reboot task after this one and ensures all updates for that category
+    is installed in one go.\r\nAsync does not work when reboot=true.\r\n\r\nDefault
+    is 'false'"
+  options: "false\r\ntrue"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: reboot_timeout
+  required: false
+  input_type: user
+  description: "The time in seconds to wait until the host is back online from a reboot.\r\nThis
+    is only used if reboot=true and a reboot is required.\r\n\r\nDefault is '1200'"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: server_selection
+  required: false
+  input_type: user
+  description: "Defines the Windows Update source catalog.\r\ndefault Use the default
+    search source. For many systems default is set to the Microsoft Windows Update
+    catalog. Systems participating in Windows Server Update Services (WSUS), Systems
+    Center Configuration Manager (SCCM), or similar corporate update server environments
+    may default to those managed update sources instead of the Windows Update catalog.\r\nmanaged_server
+    Use a managed server catalog. For environments utilizing Windows Server Update
+    Services (WSUS), Systems Center Configuration Manager (SCCM), or similar corporate
+    update servers, this option selects the defined corporate update source.\r\nwindows_update
+    Use the Microsoft Windows Update catalog.\r\n\r\nDefault is 'default'"
+  options: "default\r\nmanaged_server\r\nwindows_update"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: state
+  required: false
+  input_type: user
+  description: "Controls whether found updates are downloaded or installed or listed\r\nThis
+    module also supports Ansible check mode, which has the same effect as setting
+    state=searched\r\n\r\nDefault is 'searched'"
+  options: "installed\r\nsearched\r\ndownloaded"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
+  hidden_value: false
+- name: use_scheduled_task
+  required: false
+  input_type: user
+  description: "Will not auto elevate the remote process with become and use a scheduled
+    task instead.\r\nSet this to true when using this module with async on Server
+    2008, 2008 R2, or Windows 7, or on Server 2008 that is not authenticated with
+    basic or credssp.\r\nCan also be set to true on newer hosts where become does
+    not work due to further privilege restrictions from the OS defaults."
+  options: "false\r\ntrue"
+  advanced: false
+  value_type: plain
+  resource_type: Katello::ActivationKey
```

Comment:
I don't think that the resource_type should be set like this.

Reply:
Fixed in second commit